internal/host/validator.go

```diff
@@ -414,6 +401,19 @@ func (v *validator) getDiskEncryptionForDay2(host *models.Host) (*types.Luks, er
 	return &config.Storage.Luks[0], nil
 }
 
+func (v *validator) areTangServersReachable(c *validationContext) ValidationStatus {
+	if c.host.TangConnectivity == "" {
```

Comment:
How do we differ this case from the case in which an old agent that doesn't run the tang validation doesn't send the info?

The fact that this function is missing a flow in which we return `ValidationSuccessSuppressOutput` make me suspicious :)

Reply:
Thanks for the comment!
I moved the code that handled "no such action" to `areTangServersReachable()` and used `ValidationSuccessSuppressOutput`.

I re-tested e2e both positive and negative cases and it looks good. Printing this to the log:
```go
time="2022-07-07T11:32:53Z" level=warning msg="host 44ac8510-cb48-4481-9284-d8854033da48 replied to StepType: tang-connectivity-check with: failed to find action for step type tang-connectivity-check. Validation will pass to keep backward compatibility with discovery agent version: latest" func="github.com/openshift/assisted-service/internal/host.(*validator).areTangServersReachable" file="/go/src/github.com/openshift/origin/internal/host/validator.go:413" pkg=host-state
```